### ISSUE TITLE
Anaconda / Windows setup issues?

### ISSUE THREAD

**Issue by bdeboe** [caretdev/irissqlcli#2]
This has likely to do with my setup, but if you have suggestions that could save me (and possibly others) on what to look for to get past this and save time, I'm all ears!

```
PS C:\Users\bdeboe\GitHub\bdeboe> irissqlcli iris://localhost:51775/USER
Traceback (most recent call last):
  File "C:\Users\bdeboe\Anaconda3\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\bdeboe\Anaconda3\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\bdeboe\Anaconda3\Scripts\irissqlcli.exe\__main__.py", line 4, in <module>
  File "C:\Users\bdeboe\Anaconda3\lib\site-packages\irissqlcli\__init__.py", line 2, in <module>
    from . import main
  File "C:\Users\bdeboe\Anaconda3\lib\site-packages\irissqlcli\main.py", line 38, in <module>
    from irissqlcli.completion_refresher import CompletionRefresher
  File "C:\Users\bdeboe\Anaconda3\lib\site-packages\irissqlcli\completion_refresher.py", line 2, in <module>
    from .packages.special.main import COMMANDS
  File "C:\Users\bdeboe\Anaconda3\lib\site-packages\irissqlcli\packages\special\__init__.py", line 11, in <module>
    from . import dbcommands
  File "C:\Users\bdeboe\Anaconda3\lib\site-packages\irissqlcli\packages\special\dbcommands.py", line 8, in <module>
    from sqlite3 import ProgrammingError
  File "C:\Users\bdeboe\Anaconda3\lib\sqlite3\__init__.py", line 57, in <module>
    from sqlite3.dbapi2 import *
  File "C:\Users\bdeboe\Anaconda3\lib\sqlite3\dbapi2.py", line 27, in <module>
    from _sqlite3 import *
ImportError: DLL load failed while importing _sqlite3: The specified module could not be found.
```

**Comment by bdeboe** [comment]
running Python 3.9.13

Also: the slim Docker image works smoothly. Nice work!!

**Comment by daimor** [comment]
Interesting, I'll try to check what's going on there

**Comment by daimor** [comment]
released 0.2.0, and I believe that it should be fixed too, could you test it again?

**Comment by bdeboe** [comment]
yes, works fine on Windows for me now, including autocomplete 👍 👍
